app/views/admin/school_groups/_onboarding_schools.html.erb

```diff
@@ -21,7 +21,7 @@
           </td>
           <td class="nowrap text-right">
             <% if onboarding.school %>
-              <%= link_to 'Issues', admin_school_issues_path(onboarding.school), class: "btn btn-sm" %>
+              <%= render 'admin/school_groups/issues_link', path: admin_school_issues_path(onboarding.school), issues: onboarding.school.issues.issue.status_open.count, notes: onboarding.school.issues.note.count %>
```

Comment:
very (very) minor thing but maybe the keyword arguments could be renamed `issues_count:` and `notes_count:` here? 

```<%= render 'admin/school_groups/issues_link', path: admin_school_issues_path(onboarding.school), issues_count: onboarding.school.issues.issue.status_open.count, notes_count: onboarding.school.issues.note.count %>```

(and the same in `issues_link.html.erb`, `_active_schools.html.erb` and `_removed_schools.html.erb`)

Reply:
Great idea, I'll do that, thanks @iant 👍